skills/test-plan-case-implement/SKILL.md

```
@@ -108,34 +108,34 @@ If `odh_test_context_path == "null"`, ask user via AskUserQuestion:
+```
 
-Store: `code_repo` (e.g., `opendatahub-io/notebooks`)
+The script extracts `--target-repo` value from `$ARGUMENTS`, validates it, and returns the repo name or path.
+If validation fails, the script exits with error.
 
-#### 0.5 Locate code repository
+#### 0.5 Locate target repository
 
-Find code repo locally:
+Find target repo locally:
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)